app-hw-37/src/CardsList/Card.tsx

```diff
@@ -45,11 +45,13 @@ export const Card = ({ cardinfo, showFullScreen }: Cardprops) => {
 				</div>
 			</Link>
 			<div className='post__marks'>
-				<Likes postId={cardinfo.id} postLikes={cardinfo.likes} postsDislikes={cardinfo.dislikes}></Likes>
+				<Likes postId={cardinfo.id}></Likes>
 				<div className='post__favmark-wrapper'><Mark style={favourite ? { fill: 'green' } : { fill: 'black' }} onClick={() => {
 					favourite ? dispatch(removeFromFavourites(cardinfo)) : dispatch(setFavourites(cardinfo))
 				}} /></div>
 			</div>
 		</div>
 	)
-}
\ No newline at end of file
+}
+
+export { Mark }
```

Comment:
I'm not sure about such export
Looks complex
Possible solution - put all svg in one place and add file with there name export. 
But in some cases your way can be best solution